src/Mealz/MealBundle/Resources/translations/general.de.yml

```diff
@@ -79,3 +79,6 @@ form:
     title: Titel
     limit: Limit
     order: Sortierung
+
+  week:
+    notification: Kunden benachrichtigen
```

Comment:
Ich würde das gerne anders benennen. In der Richtung: Menü in Mattermost veröffentlichen